__mocks__/@tsndr/cloudflare-worker-jwt.ts

```diff
@@ -0,0 +1,14 @@
+"use strict";
+import jsonwebtoken from "jsonwebtoken";
```

Comment:
why do we need this package? It is not being used anywhere so why are we mocking it?

Reply:
yes we did it afterwards